packages/serialization/multipart/src/multipartSerializationWriter.ts

```diff
@@ -42,7 +42,9 @@ export class MultipartSerializationWriter implements SerializationWriter {
       }
       this.writeRawStringValue(value);
     }
-    this.writeRawStringValue("\r\n");
```

Comment:
do you think removing this line instead, and adding `\r\n` to the calls to writeStringValue that require it would be a viable alternative here?

Reply:
can't we make writeRawStringValue public?
if so I could replace this line with:
writeRawStringValue

Reply:
ideally we wouldn't. The idea is that the Serializer implementations only expose the methods defined in the interfaces.